### ISSUE TITLE
SFTP support

### ISSUE THREAD

**Issue by graille** [subdavis/Tusk#291]
### A feature you'd like to see
Support of SFTP to get database files

### Alternatives you've considered
WebDAV, but for security and access reasons, I can't deploy a good WebDAV server on my server

### Additional remarks
I can develop it. I just want to be sure you could be interested in this kind of feature. 
According to #172 it seems you were, but are you still ?

**Comment by subdavis** [comment]
Still interested, but currently unavailable for other work as the vuex branch consumes all my time.

Any major efforts right now on other features would either 

A) need to be totally rewritten once vuex is done or
B) go in the trash bin once that rewrite is done because rebasing would be almost impossible.

**Comment by subdavis** [comment]
Closing as duplicate of #172
